Script/AtomicWebViewEditor/interop.ts

```diff
@@ -213,6 +218,22 @@ export default class HostInteropType {
     resourceDeleted(path: string) {
         editorCommands.resourceDeleted(path);
     }
+
+    /**
+     * Host is notifying client that there are preferences to load and passing us the path
+     * of the prefs.
+     * @param  {string} prefUrl
+     */
+    loadPreferences(prefUrl: string) {
+        console.log("Load preferences called for :" + prefUrl);
```

Comment:
The log is starting to have a fair bit of output in it.  So, we should probably start trying to catch debugging code before landing,  A way to set debug level logging would be good, we already have this native side, just needs to be wrapped.